components/sysutils/gptfdisk/files/Makefile.solaris

```diff
@@ -0,0 +1,57 @@
+CC=gcc
```

Comment:
Spote one minor issue. Would be better to use CC ?= ... and pass CC and CXX variables in the environment